server/raw/jsfiddle/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1">
+    <title>jsfiddle fullscreen Tester | Fomantic-UI</title>
+    <link rel="stylesheet" type="text/css" href="/dist/semantic.min.css" >
+    <meta name="description" content="Testing jsfiddle fullscreen in Fomantic-UI. Especially for mobile" />
+    <meta name="keywords" content="jsfiddle, fullscreen, fomantic, semantic, mobile, html5, ui, library, framework, javascript, fork, official" />
+</head>
+<style>
+    body {
+        display: flex;
+        flex-direction: column;
+    }
+    iframe {
+        flex-grow:1;
+    }
+    @media only screen and (min-width: 767px) {
+        [class*="mobile only"] {
+            display: none !important;
+        }
+    }
+</style>
+<body>
+    <div class="ui top attached basic segment" >
+        <h3 class="ui header">
+            <a class="ui image" href="/"><img src="/images/logo.png"></a>
+            <div class="content">
+                jsfiddle tester <span class="mobile only">for mobile</span>
+            </div>
+        </h3>
+        <i class="small exclamation triangle icon"></i>
+        The Code below is located at <a id="orgfiddle" href="">https://jsfiddle.net</a> and embedded via an iframe
+    </div>
+    <iframe id="jsfiddle" src="http://" frameborder="0"></iframe>
+</body>
+<script>
+    var jsfiddleInfo = window.location.href.split("#!");
+    if(jsfiddleInfo.length > 1){
+        var orgFiddle = '//jsfiddle.net/'+jsfiddleInfo[1].replace(/[^a-zA-Z0-9\/]/g,'');
+        document.getElementById('jsfiddle').src = orgFiddle+'/embedded/result/';
+        document.getElementById('orgfiddle').href = orgFiddle;
+    }
+    else {
+        window.location.href = '/';
+    }
```

Comment:
Can we remove this part? This will redirect the page in an infinite loop if there's no fiddle URL.

This URL (https://fomantic-ui.com/jsfiddle/#!lubber/wt0heq5k/99) works. But, this URL (https://fomantic-ui.com/) will cause infinite redirect.

Reply:
No, it doesn't. It will redirect to the root of the website rather than the root of the jsfiddle folder. So, if no jsfiddle is given the url https://fomantic-ui.com/jsfiddle will redirect once to https://fomantic-ui.com

Reply:
The index.html is inside the jsfiddle folder 😉

Reply:
Well. It happens on my PC running with Nginx server. Maybe, because of my Nginx configuration, I'm not sure. But, if I removed the `else` block, it doesn't happen.

Reply:
My root is the same as this HTML document on my local server. I've setup my Nginx using this document with as index.html.

So, this definitely cause infinite redirect. Not sure, how would this will be hosted in production though.

I'm not running this using FUI document static site generator due to this issue (https://github.com/fomantic/Fomantic-UI-Docs/issues/192).

Reply:
mmh, i did that too.
I point the root of vhost to the root of the gh-pages branch of the docs repo
my nginx config
```nginx
server {
        listen       999;
        server_name  localhost;
		root    c:/gitrepos/Fomantic-UI-Docs;
		index   index.htm index.html;
    }
```
So when i point to localhost:999/jsfiddle it redirects to localhost:999 as expected 

`/` is always the domains root, so inside of the jsfiddle folder a redirect to `/` should never point to its own folder (for that the redirect should be `./` (leading dot) instead.

Of course we can remove this part, but i am curious why your setup behaves this way.... Whats your nginx config?

Reply:
the jsfiddle folder is not the root of the docs domain, you if you want to simulate the docs on your local nginx config, checkout the gh-pages branch, create a jsfiddle folder , put the index.html from this PR into this subfolder. 
Of course, if your localhost root is the jsfiddle folder itself, then its an indefinite redirect, but thats not how it is available/created on the fomantic domain/gh-pages branch

Reply:
My nginx config is almost the same as yours.
```
server {
  listen 80;

  server_name ui.frontend ~(?<vendor>.+)\.ui.frontend$;

  root /var/www/front-end/ui/$vendor;

  index index.php index.html index.htm;

  location / {
    try_files $uri $uri/ /index.html;
  }

  error_log /var/log/nginx/front-end/ui/error.log;
  access_log /var/log/nginx/front-end/ui/access.log;
}
```
I placed this test file inside the directory 'fui' as index.html, and my virtual address is `fui.ui.frontend`.

But, as you explained, this document will be placed inside the 'jsfiddle' directory in real life. So, I don't see it as problem. But, for different set-up and test, it won't work.

Reply:
For testing, you should always try to create a setup equal to the real world setup 😄 .
So, lets see what the other maintainers say, before i am removing this part. I just wanted to  fetch the situation when no jsfiddle code is given

Reply:
I can't setup and build the DOC repo with docpad with my current node environment. Both my desktop PC and laptop have problem using docpad :cry: as reported in this issue (https://github.com/fomantic/Fomantic-UI-Docs/issues/192).

Reply:
@ko2in @hammy2899 may help? He is always doing the docs builds.

Reply:
@ko2in Does this help (uninstall globally / reinstalling specific docpad version)?
https://github.com/Semantic-Org/Semantic-UI-Docs/issues/404#issuecomment-486644651

Reply:
@lubber-de Thank you. With docpad@6.79.4, it seems working and I can build the docs and see it running in the browser.

But, the extension eco doesn't seem to be working properly. It returns some error in the console as below:
```
Error: Rendering the extension "eco" to "html" on "/home/koko/Desktop/fui/docs/server/partials/ui-javascript.html.eco" didn't do anything.
```

And I've to avoid `docpad update` and `docpad upgrade` command as mentioned in [README.md](https://github.com/fomantic/Fomantic-UI-Docs#installing-dependencies). Otherwise, it'll install the latest docpad locally in the project directory, and failed to deal with docpad again.

Reply:
Yeah, unfortunately I think we have to live with this until we redesign the doc pages using a different static site generator as mentioned in #319

Reply:
This redirect is ok for me, since I am using docpad specific for this repo.